google-cloud-spanner/src/main/java/com/google/cloud/spanner/Mutation.java

```diff
@@ -402,20 +404,43 @@ private boolean isFloat32NaN(Value value) {
     return value.getType().equals(Type.float32()) && Float.isNaN(value.getFloat32());
   }
 
-  static void toProto(Iterable<Mutation> mutations, List<com.google.spanner.v1.Mutation> out) {
+  /**
+   * Converts the list of mutations to the corresponding protobuf mutations and returns a random
+   * mutation from the available list based on the following heuristics:
+   *
+   * <p>1. Prefer mutations other than INSERT, as INSERT mutations may contain autogenerated columns
```

Comment:
nit: (feel free to leave as is, as this is not a public method, but for future reference) when making a summation in Javadoc, then you should do this:

```
/**
 * <ol>
 *   <li>Item 1</li>
 *   <li>Item 2</li>
 * </ol>
 */
```

Reply:
Done